src/arithmetic/curves.rs

```diff
@@ -60,7 +60,7 @@ pub trait Curve:
     /// Obtains the additive identity.
     fn zero() -> Self;
 
-    /// Obtains the base point of the curve.
+    /// Obtains the base point of the curve, if defined.
```

Comment:
Let's revert this back to the original comment. The "if defined" contradicts the API contract in the group crate (which we aren't using yet but will) and since we're not exposing the isogeneous curve in the API it doesn't matter anyway.

Reply:
```suggestion
    /// Obtains the base point of the curve.
```